src/utils/pgwire/src/pg_protocol.rs

```diff
@@ -333,12 +334,34 @@ where
 
     async fn process_query_msg(&mut self, query_string: io::Result<&str>) -> PsqlResult<()> {
         let sql = query_string.map_err(|err| PsqlError::QueryError(Box::new(err)))?;
-        tracing::trace!(
-            target: "pgwire_query_log",
-            "(simple query)receive query: {}", sql);
-
+        let start = Instant::now();
         let session = self.session.clone().unwrap();
+        let session_id = session.id().0;
+        let res = self.inner_process_query_msg(sql, session).await;
+        // Record query run successfully
+        let mills = start.elapsed().as_millis();
+        let truncated_sql = &sql[..std::cmp::min(sql.len(), 1024)];
+        match res {
+            Ok(x) => {
+                tracing::trace!(
+                    target: "pgwire_query_log",
+                    "(simple query) session: {}, status: ok, time: {}ms,  sql: {}", session_id, mills, truncated_sql);
```

Comment:
What about using _structured fields_ for these properties? So that we may be able to analyze these query logs more easily.

https://docs.rs/tracing/latest/tracing/index.html#recording-fields